src/main/java/gov/usgs/cida/pubs/busservice/xml/XmlBusService.java

```diff
@@ -134,13 +154,28 @@ public static String getDocumentHtml(URL xmlDoc, File xlsStylesheet, boolean val
 		return htmlBytes.toString();
 	}
 
-	private static void updateImageLinks(URL xmlUrl, Node node) {
+	// make sure the image link to add to the html is a valid Url
```

Comment:
I don't think we should be validating the environment/configuration variables. They should be taken as is and if wrong, the application allowed to malfunction. That way any problems will hopefully be caught soon after a fresh deployment.